staging/src/k8s.io/kubectl/pkg/explain/v2/explain_test.go

```diff
@@ -19,133 +19,63 @@ package v2
-var apiGroupsGVR schema.GroupVersionResource = schema.GroupVersionResource{
+var apiGroupsPath = "apis/discovery.k8s.io/v1"
+
+var apiGroupsGVR = schema.GroupVersionResource{
 	Group:    "discovery.k8s.io",
 	Version:  "v1",
 	Resource: "apigroups",
 }
-var apiGroupsDocument map[string]interface{} = func() map[string]interface{} {
-	var doc map[string]interface{}
-
-	err := json.Unmarshal([]byte(apiDiscoveryJSON), &doc)
-	if err != nil {
-		panic(err)
-	}
-
-	return doc
-}()
-
-type FakeOpenAPIV3Client struct {
-	// Path:
-	//		ContentType:
-	//			OpenAPIV3 Schema bytes
-	Values      map[string]map[string][]byte
-	FetchCounts map[string]map[string]int
-	lock        sync.Mutex
-}
-
-type FakeGroupVersion struct {
-	Data        map[string][]byte
-	FetchCounts map[string]int
-	Lock        *sync.Mutex
-}
-
-func (f *FakeGroupVersion) Schema(contentType string) ([]byte, error) {
-	f.Lock.Lock()
-	defer f.Lock.Unlock()
-
-	if count, ok := f.FetchCounts[contentType]; ok {
-		f.FetchCounts[contentType] = count + 1
-	} else {
-		f.FetchCounts[contentType] = 1
-	}
-
-	data, ok := f.Data[contentType]
-	if !ok {
-		return nil, errors.New("not found")
-	}
-	return data, nil
-}
-
-func (f *FakeOpenAPIV3Client) Paths() (map[string]openapi.GroupVersion, error) {
-	if f.Values == nil {
-		return nil, errors.New("values is nil")
-	}
-
-	res := map[string]openapi.GroupVersion{}
-	if f.FetchCounts == nil {
-		f.FetchCounts = map[string]map[string]int{}
-	}
 
-	for k, v := range f.Values {
-		counts, ok := f.FetchCounts[k]
-		if !ok {
-			counts = map[string]int{}
-			f.FetchCounts[k] = counts
-		}
-		res[k] = &FakeGroupVersion{Data: v, FetchCounts: counts, Lock: &f.lock}
-	}
-	return res, nil
-}
+var openAPIV3SpecDir = "../../../../../../../api/openapi-spec/v3"
```

Comment:
cross-module paths like this don't work well, I think this broke the publishing bot when testing modules before pushing them. cc @dims @nikhita

Reply:
I will revert

Reply:
opened revert at https://github.com/kubernetes/kubernetes/pull/115785 so we don't have broken dependencies for the alpha.2 tag being cut tonight